pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -73,6 +73,12 @@ const (
 	defaultIOThread            = uint(1)
 	bootMenuTimeoutMS          = uint(10000)
 	QEMUSeaBiosDebugPipe       = "/var/run/kubevirt-private/QEMUSeaBiosDebugPipe"
+	kvmPath                    = "/dev/kvm"
+)
+
+var (
+	errKVM   = errors.New("unable to use kvm device")
+	probeKVM = probeKVMDevice
```

Comment:
Thank you for your PR. I'm sorry for not noticing your PR before.

Adding a global variables is a code smell, even if they are modified only during tests. Unit tests should not use side channels to pass information to the tested function.
https://github.com/kubevirt/kubevirt/pull/16295 attempts to avoid that by adding this as another field to the converter context struct https://github.com/kubevirt/kubevirt/pull/16295/files#diff-68e1ada047efdbcbc2c2c5ebfff4be524dddd2a1c1fa1097a42c88e931b06211R634.

Admittedly, this struct is so big, that making it bigger is not a great practice on its own right. Still, I believe it is better than adding a global variables just for this.